tools/install_avoidance.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
FWIW Don't know what is _standard_, but generally I quite like it if scripts have brief docs about what they do/when they are used. Maybe a link to where this is defined.

Entirely up to you